README.md

```diff
@@ -13,17 +13,7 @@ service.
 
 ## Getting Started
 
-* Try the [sample app](Sample).
-  * Using [CocoaPods](https://cocoapods.org):
```

Comment:
'pod try' should present the developer with a menu to choose a sample app (we should test), so I think we can keep this.

Reply:
Per our conversation, I've elected to update the `.cocoapods.yml` file to point to the existing ObjC sample as canonical.